adapters/mqtt-vertx-base/src/test/java/org/eclipse/hono/adapter/mqtt/AbstractVertxBasedMqttProtocolAdapterTest.java

```diff
@@ -712,16 +719,19 @@ public void testUploadEmptyCommandResponseSucceeds(final VertxTestContext ctx) {
         final ResourceIdentifier address = ResourceIdentifier
                 .fromString("command/my-tenant/4712/res/1010f8ab0b53-bd96-4d99-9d9c-56b868474a6a/200");
         adapter.uploadCommandResponseMessage(newMqttContext(messageFromDevice, endpoint), address)
-                .setHandler(ctx.completing());
+                .setHandler(ctx.succeeding(result -> {
+                    verify(sender).sendCommandResponse(any(), any());
```

Comment:
this needs to be wrapped in `ctx.verify(Handler<Void>)`

Reply:
Right. Sorry. Done that.